geode-apis-compatible-with-redis/README.md

```diff
@@ -329,9 +329,10 @@ start server \
 |                    	|                                                    	| ZREVRANK                      	|
 |                    	|                                                    	| ZSCAN                         	|
 |                    	|                                                    	| ZSCORE                        	|
-|                    	|                                                    	| ZUNIONSTORE                   	|                   	|
+|                    	|                                                    	| ZUNIONSTORE                   	|
 
 **NOTES:**
 
 [1] - UNLINK is implemented as a synonym to DEL and does not unlink asynchronously.
 [2] - SLOWLOG is implemented as a NoOp.
+[3] - Redis accepts Longs as the parameters for the HSCAN command. The Geode APIs compatible with Redis only accept Integer values. Values out of the range of the 32-bit integer type will throw an exception.
```

Comment:
I'm torn by the wording of this.  We return an error type that follows Redis's [RESP protocol](https://redis.io/topics/protocol).  The client library actually raises the exception when an error reply is received, not us.  In this scenario, Redis seems to just say "an error is returned". The SCAN documentation is a bit different, but if you look at [DECRBY](https://redis.io/commands/DECRBY) they say the following:
```
An error is returned if the key contains a value of the wrong type or contains a string that can not be represented as integer. This operation is limited to 64 bit signed integers.
```

Also, maybe we could say `Redis accepts 64-bit signed integers for the cursor and COUNT parameters for the HSCAN command.`  and say we accept 32-bit integer values instead of `Integer values` to be a bit more specific (I see that specification in the following sentence, but thought it might be helpful to distinguish the value we say we accept in the previous sentence).